### PR TITLE
For Jetpack Sharing Buttons block add learn more link

### PR DESCRIPTION
**Pull request by andrii-lysenko**
Related to https://github.com/Automattic/vulcan/issues/141

## Proposed Changes
For new Sharing Buttons block we want to have similar support pages and Learn More link as for 'Social List' block

## Testing Instructions

I couldn't find a way to test it locally. Using `wp-env` it doesn't include jetpack to test block there. 
It seems as pretty easy change, though if there's a way to test it, I would appreciate guidance.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts [hunk 1]
```diff
@@ -192,6 +192,9 @@ const blockLinks: { [ key: string ]: string } = {
 	'jetpack/mailchimp': 'https://wordpress.com/support/wordpress-editor/blocks/mailchimp-block/',
 
 	'jetpack/paywall': 'https://wordpress.com/support/paid-newsletters/#use-the-paywall-block',
+
+	'jetpack/sharing-buttons':
+		'https://wordpress.com/support/wordpress-editor/blocks/sharing-buttons-block/',
```
Comment (tyxla) [inline apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts]: The link doesn't seem to work. Do we need to create this support article first?

**Review by tyxla — COMMENTED** [review]
Code-wise this looks good, but we'll need to make sure the link is valid and leads to the expected support article page.

File: apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts (hunk 1 above)
Comment (andrii-lysenko) [reply apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts]: Thanks for the review. The documentation in general is still not ready, but we would make sure that the feature will go live after we have one (task https://github.com/Automattic/vulcan/issues/142).
This is a part of Call For Testing pdtkmj-2dQ-p2.

**Review by jeherve — APPROVED** [review]
I created and published a first version of the support document. This should be ready to merge.

**Pull request merged**